packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -151,51 +111,36 @@ export class PlaygroundWorkerEndpoint extends PHPWorker {
 		};
 	}
 
-	async getSupportedWordPressVersions() {
+	async getMinifiedWordPressVersions() {
 		return {
-			all: SupportedWordPressVersions,
-			latest: LatestSupportedWordPressVersion,
+			all: MinifiedWordPressVersions,
+			latest: LatestMinifiedWordPressVersion,
 		};
 	}
 
-	async resetVirtualOpfs() {
-		if (!virtualOpfsRoot) {
-			throw new Error('No virtual OPFS available.');
-		}
-		await virtualOpfsRoot.removeEntry(virtualOpfsDir!.name, {
-			recursive: true,
-		});
-	}
-
-	async reloadFilesFromOpfs() {
-		await this.bindOpfs({
-			opfs: lastOpfsHandle!,
-			mountpoint: lastOpfsMountpoint!,
-		});
+	async hasOpfsMount(mountpoint: string) {
+		return mountpoint in this.unmounts;
 	}
 
-	async bindOpfs(
-		options: Omit<BindOpfsOptions, 'php' | 'onProgress'>,
+	async mountOpfs(
+		options: MountDescriptor,
 		onProgress?: SyncProgressCallback
 	) {
-		lastOpfsHandle = options.opfs;
-		lastOpfsMountpoint = options.mountpoint;
-		await bindOpfs({
-			php: this.__internal_getPHP()!,
-			onProgress,
-			...options,
-		});
-	}
-
-	async journalFSEvents(
-		root: string,
-		callback: (op: FilesystemOperation) => void
-	) {
-		return journalFSEvents(this.__internal_getPHP()!, root, callback);
+		const php = this.__internal_getPHP()!;
+		this.unmounts[options.mountpoint] = await php.mount(
+			options.mountpoint,
+			createDirectoryHandleMountHandler(options.handle, {
+				initialSync: {
+					onProgress,
+					direction: options.initialSyncDirection,
+				},
+			})
+		);
 	}
 
-	async replayFSJournal(events: FilesystemOperation[]) {
-		return replayFSJournal(this.__internal_getPHP()!, events);
+	async unmountOpfs(mountpoint: string) {
+		this.unmounts[mountpoint]();
+		delete this.unmounts[mountpoint];
```

Comment:
I mounted a directory with only one PHP file and when I clicked sync this threw an error because `this.unmounts[mountpoint]` didn't exist. 

```suggestion
		if (this.unmounts[mountpoint]) {
			this.unmounts[mountpoint]();
			delete this.unmounts[mountpoint];
		}
```

Reply:
I think we would need to avoid the issue that led to this one rather than doing nothing if an expected unmount function does not exist.

I am not able to reproduce this issue with the latest in this PR, so I will resolve this for now.